target/linux/brcm47xx/image/Makefile

```diff
@@ -815,23 +917,29 @@ endef
 TARGET_DEVICES += linksys-e4200-v1
 
 define Device/netgear-wgr614-v10-na
-  DEVICE_TITLE := Netgear WGR614 v10 North America
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := WGR614
+  DEVICE_VARIANT := v10 North America
   $(Device/netgear)
   NETGEAR_BOARD_ID := U12H139T01_NETGEAR
   NETGEAR_REGION := 2
 endef
 TARGET_DEVICES += netgear-wgr614-v10-na
 
 define Device/netgear-wgr614-v10
-  DEVICE_TITLE := Netgear WGR614 v10
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := WGR614
+  DEVICE_VARIANT := v10
```

Comment:
Here one could add a distinction from North America (e.g. "ROW" rest of world or similar)
Again, this not a request for change, but an opportunity if someone has a good idea.

Reply:
I do not have an opinion here. If you like to have, I will put it in.

Reply:
Maybe personal, but I dislike the binarity of "north america" vs "rest of the world". I guess just leave it as is :)

Reply:
I borrowed that from the Steam store, where you occasionally have versions of games which are specifically adjusted for a certain country, e.g. by removing Third Reich symbols in Germany. Those are then labelled "DE" for Germany and "ROW" for the other countries.
Another frequently used identifier is "INT" for international.

However, I think choosing between "v10" and "v10 (NA)" will also be obvious enough ...